lib/features/home/presentation/view/home_page.dart

```diff
@@ -25,7 +25,7 @@ class _HomePageState extends State<HomePage> {
         ),
       ),
       body: Padding(
-        padding: const EdgeInsets.only(left: 10.0),
+        padding: const EdgeInsets.only(left: 8.0),
```

Comment:
The ListView widget can take this padding directly. You don't have to wrap it with a Padding widget